### ISSUE TITLE
[TF2] UI Scaling Crops messes up some parts of the ui

### ISSUE THREAD

**Issue by TheSillyDoggo** [ValveSoftware/Source-1-Games#6419]
Monitor Resolution: 2560x1440

Open Source Attributions Page:
![image](https://github.com/user-attachments/assets/cd93ae12-54fe-4684-a6c2-7ed0c2fee629)

Bug Reporter Page:
![image](https://github.com/user-attachments/assets/e73c4df3-1c0a-4b69-b82d-f08877c7a080)

Developer Commentary Thumbnails:
![image](https://github.com/user-attachments/assets/8f8b9785-bb3e-4867-b27b-cce8322ae01a)

Replays Menu:
![image](https://github.com/user-attachments/assets/c98350f7-2c97-4b16-8aa9-fd9ad7727baf)

**Comment by NULLYUKI** [comment]
Could be merged with #6418 in my opinion.

**Comment by TheSillyDoggo** [comment]
> Could be merged with #6418 in my opinion.

yeah i dont really use github issues often so idk how to like properly do them and stuff lol

**Comment by TheSillyDoggo** [comment]
![image](https://github.com/user-attachments/assets/382c1c67-ea68-45ed-b636-4b0432bc2ed1)
kill feed has extra padding on the left and right

**Comment by chl0rine03** [comment]
can't see ranks in casual anymore too and right clicking items in inventory is busted as well as the menu that appears is far too large

**Comment by groovyvs** [comment]
any fix yet? my hud is so messed up by this that i cant even queue casual without completely crashing

**Comment by TheSillyDoggo** [comment]
![image](https://github.com/user-attachments/assets/c21b0cc3-8953-40b1-8b01-badf937aa39f)
The text in the engineer (hud?) appears to be bigger than usual, however this may just be because of the font being higher quality

**Comment by jh34ghu43gu** [comment]
> any fix yet? my hud is so messed up by this that i cant even queue casual without completely crashing

This sounds like a custom hud issue and you'd want to take that up with the hud creator.

**Comment by Ashetf2** [comment]
Yeah, custom hud issue.

Btw, this has not been updated. Bink is no longer used in TF2 afaik

![imagen](https://github.com/user-attachments/assets/51158867-b11d-46d0-930f-f28e10dcccad)

**Comment by TheSillyDoggo** [comment]
not a custom hud issue, i do not use a custom hud

**Comment by jh34ghu43gu** [comment]
> not a custom hud issue, i do not use a custom hud

We were responding to the guy I quoted who is crashing trying to queue.

**Comment by groovyvs** [comment]
> any fix yet? my hud is so messed up by this that i cant even queue casual without completely crashing

Ended up fixing this issue by removing a tf_english.txt file in my hud that was broken by the update

**Comment by RiverHornet3209** [comment]
> Bink is no longer used in TF2 afaik

The 32-bit version of TF2 uses Bink
